openeo/rest/datacube.py

```diff
@@ -795,15 +813,24 @@ def _polygonal_timeseries(self, polygon: Union[Polygon, MultiPolygon, str], func
                 }
             }
 
+        reducer = {"process_graph": PGNode(
+                    process_id=func,
+                    arguments={"data": {"from_parameter": "data"}}
+                )}
+        # TODO: Might be necessary for 1.0.0rc2
```

Comment:
same as above: from my understanding of spec this additional "process" level is not necessary